Makefile

```diff
@@ -31,9 +31,7 @@ version.properties:
 build: $(TARGET)
 
 $(TARGET): .target_os $(SOURCES)
-	$(if $(shell command -v dep 2> /dev/null),$(info Found golang/dep),$(error Please install golang/dep))
-	dep ensure
-	GOOS=$(OS) GOARCH=${ARCH} CGO_ENABLED=0 go build -a -installsuffix cgo --ldflags '-w' -o $@ 
+	GOOS=$(OS) GOARCH=${ARCH} CGO_ENABLED=0 go build -a --ldflags '-w' -o $@ ./cmd/$@
```

Comment:
does `make test` still work? just want to make sure the macros above don't need to be changed

Reply:
Works locally + is part of the travis tests